docs/monitor/metrics.md

```diff
@@ -89,3 +89,12 @@ Since v1.5.0, you can use the following metrics to replace:
 - `kyuubi.operation.total.ExecuteStatement`
 - `kyuubi.operation.opened.ExecuteStatement`
 - `kyuubi.operation.failed.ExecuteStatement.${errorType}`
+
+Since v1.6.0, you can use the following metrics to get the current state counter for a specified operation type:
```

Comment:
the doc is eligible. please follow the table above. the one you added can only be an extra hint.